### ISSUE TITLE
already detected by windows defender 

### ISSUE THREAD

**Issue by xbeatzsec** [t3l3machus/hoaxshell#37]
When the script is executed in powershell the windows defender pops up this red flag, even with -o (--obfuscate) option

![image](https://user-images.githubusercontent.com/75812403/197831241-cff79157-d7b5-4808-b34c-b3faaea68da2.png)

**Comment by t3l3machus** [comment]
yeah, i know, i will update the auto obf when i can. Obfuscate manually it's pretty simple
